frontend/src/lib/components/Support/SupportForm.tsx

```diff
@@ -129,6 +131,8 @@ export function SupportForm(): JSX.Element | null {
                     disabledReason={
                         !user
                             ? 'Please login to your account before opening a ticket unrelated to authentication issues.'
+                            : !hasAvailableFeature(AvailableFeature.EMAIL_SUPPORT)
```

Comment:
This will work but it feels like this logic check should be tied to the same one in the `openSupportForm` listener and in the SidePanelSupport.tsx - maybe an action they pull from? 

I say this because, this should only be disabled if the `target_area` is set and they don't have support on their plan. If there is an edge case where they don't have support but they see the form, the input will be disabled, and they won't be able to submit it (i.e. the logic in SidePanelSupport.tsx is changed and these other two locations aren't updated).

Not a blocker though, just a suggestion, so I'll let you decide if it's worth the change.

Reply:
I agree that it'd be better if the checks were consistent.

Reply:
The issue here would be that users could open the `emailForm` on the billing page and edit the `target_area` on another page since we are keeping the menu open currently.